src/test/EigenPod.t.sol

```diff
@@ -366,6 +397,28 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         cheats.stopPrank();
     }
 
+    function testWithdrawNonBeaconChainETHBalanceWei() public {
+        IEigenPod pod = testDeployAndVerifyNewEigenPod();
+        IEigenPod _pod = eigenPodManager.getPod(podOwner);
+
+        require(_pod == pod, "add check");
```

Comment:
don't think this check is strictly necessary, since testDeployAndVerifyNewEigenPod() is returning the value of getPod() already.